test/plugin/test_out_google_cloud.rb

```diff
@@ -70,6 +70,70 @@ def test_server_error
     assert_equal 1, exception_count
   end
 
+  def test_prometheus_requests_count
+    setup_gce_metadata_stubs
+    [
+      # Single successful request.
+      [200, 1, 1, 0],
+      # Several successful requests.
+      [200, 2, 2, 0],
+      # Single failed request.
+      [401, 1, 0, 1]
+    ].each do |code, emit_count, successful_count, failed_count|
+      setup_prometheus
+      stub_request(:post, WRITE_LOG_ENTRIES_URI)
+        .to_return(status: code, body: 'Some Message')
+      (1..emit_count).each do |i|
+        d = create_driver(PROMETHEUS_ENABLE_CONFIG)
```

Comment:
As I've mentioned, I have to create a new driver for each request I want it to issue, since otherwise all entries get bundled to a single request

I would really appreciate if you could advise how to make it issue several requests with one driver